README.md

```diff
@@ -14,12 +14,14 @@ Heroku Connect Toolbelt Plugin
 
     heroku connect:auth              - Authenticate a connection to Salesforce
     heroku connect:preauth           - Store a Salesforce authentication for future use
-    heroku connect:export            - Export a mapping configuration json file
-    heroku connect:import            - Import a mapping configuration json file
-    heroku connect:mapping MAPPING   - Returns a status JSON doc for a single mapping
-    heroku connect:setup             - configure a new connection
+    heroku connect:export            - Export a mapping configuration JSON file
+    heroku connect:import            - Import a mapping configuration JSON file
+    heroku connect:info              - Display connection information 
+    heroku connect:mapping:create    - Create a new mapping
```

Comment:
I've intentionally left `mapping:create` and `mapping:delete` undocumented for now, because they're not yet part of the documented API. I also expect the behavior of these commands to change slightly before we're done with them, so I'd rather not promise anything about them yet.

I know it kinda sucks to have them included in the output of `heroku help connect` but not documented here, though. Maybe we can add some kind of beta label to the help text in case people see them that way?